docs/integrations/sources/file.md

```diff
@@ -56,7 +56,7 @@ This choice is made possible through the optional `reader_impl` parameter.
 
 ### Limitations
 
-- Note that for local filesystem, the file probably have to be stored somewhere in the `/tmp/airbyte_local` folder with the same limitations as the [CSV Destination](../destinations/local-csv). This may not be ideal as a Source and not well tested yet.
+- Note that for local filesystem, the file probably have to be stored somewhere in the `/tmp/airbyte_local` folder with the same limitations as the [CSV Destination](../destinations/local-csv.md). This may not be ideal as a Source and not well tested yet.
```

Comment:
This actually didn't fix the issue. What fixed the issue is that you forced gitbook to resync the doc... Gitbook sometime links to branch blobs instead of the actual md file. Very annoying bug. Sometime it pops up in github sometime it doesn't